### DISCUSSION TITLE
Log Error

### DISCUSSION THREAD

**Discussion by Jomooooo** [SafeExamBrowser/seb-win-refactoring#260] [Q&A]
Hello,

Good day.

Is anyone encounter this issue?

"SEB failed to start new session! please consult log files for more information" and how did you solve it.

Appreciate your assistance.

Thank you.

**Comment by dbuechel** [comment]
This is a generic error message, and as it already states, the exact reason of failure can be found in the SEB application logs. They can be found under `%LocalAppData%\SafeExamBrowser\Logs`. Feel free to post them here so that we may help you locate your issue.
